lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -19,6 +20,12 @@ def default_decimal(obj):
         return float(obj)
 
 
+class JSONRPCException(Exception):
+    def __init__(self, err, code):
+        self.faultCode = code
+        self.faultString = err.getTraceback()
```

Comment:
better would be to keep the err: `self.err = err` and then have
```
@property
def fault_string(self):
    return err.getTraceback()
```

Generally, the dumber a constructor is the better. They should almost always be simple assignments of the passed in args to variables.